ecosystem_ci/step.json

```diff
@@ -12,7 +12,7 @@
 		]
 	},
 	"plugins": [{
-			"ray-project/dind#v1.0.10": {
+			"ray-project/dind#104e9f556a0144ddd7f6e7f6d92346dcbb75fa79": {
```

Comment:
do you have access to the dind repo? could you create a tag rather than using the commit hash?

Reply:
Sure let me do that

Reply:
@aslonnie done